taikoxyz/taiko-mono
# fix(bridge-ui): correct display of forward arrow and handling of invalid pagination input

**This pull request addresses:**
1.  A bug related to pagination in the transactions page. Previously, the forward arrow button was incorrectly displayed even beyond the last page, leading to confusion for users. This bug has been fixed to ensure that the forward arrow button is displayed only when there are additional pages to navigate to.

2. Previously, when users input a number greater than the total number of pages in the pagination input field, the application would attempt to navigate to the specified page, leading to unexpected behaviour. This commit addresses this issue by adding validation in the handleKeydown function to check if the input number exceeds the total number of pages. If the input is invalid, the application ignores the input and does nothing.

**Changes:**
- Modified the pagination logic to accurately determine when to display the forward arrow button.
- Added validation in the handleKeydown function to handle input of a number greater than the total number of pages.
- Implemented a mechanism to ignore invalid input, ensuring consistent and expected behaviour in the pagination component.

**How to Test:**
1. Navigate to the transaction records page: https://bridge.katla.taiko.xyz/transactions 
2. Verify that the forward arrow button is not displayed on the last page.
3. Verify that nothing happens if a number greater than the last page number is entered.
4. Input a number greater than the total number of pages and ensure that the forward arrow button remains hidden.


closes #16468
